pkg/bisect/bisect.go

```diff
@@ -254,14 +254,12 @@ func (env *env) bisect() (*Result, error) {
 	}
 	env.crashReport = testRes.rep
 
-	if len(cfg.Kernel.BaselineConfig) != 0 {
-		testRes1, err := env.minimizeConfig()
-		if err != nil {
-			return nil, err
-		}
-		if testRes1 != nil {
-			testRes = testRes1
-		}
+	testRes1, err := env.minimizeConfig()
+	if err != nil {
+		return nil, err
```

Comment:
- Do we want to abort the bisection attempt if config minimization fails or continue it without a minimized config? If we want to continue we can't bubble up this error here. Not sure myself what is the best choice.
- We didn't use this code path at all before. I assume once we enable this, we might see a bunch of obscure errors pop up. Maybe wrap the error in an extra sprintf here, mentioning that the error was raised during kconfig minimisation.

Reply:
> Do we want to abort the bisection attempt if config minimization fails or continue it without a minimized config? If we want to continue we can't bubble up this error here. Not sure myself what is the best choice.

The error would come from `test()` and we treat such errors as irrecoverable throughout the whole bisection code. I don't think config minimization should be any different in this respect. 

If the kernel didn't crash without those instrumentation and no problems occurred while building code/creating VMs, execution just goes on.

> Maybe wrap the error in an extra sprintf here, mentioning that the error was raised during kconfig minimisation.

Good idea, I'll adjust the code.